src/modules/Nfs.rb

```diff
@@ -385,25 +387,26 @@ def WriteOnly
           Builtins.sformat(_("Unable to create directory '%1'."), file)
         )
       end
+    end
 
-      log.debug("fstab before ordering: #{fstab}")
-      fstab = Fstab::TSort.sort(fstab)
-      log.info("fstab: #{fstab}")
-
-      SCR.Execute(
-        path(".target.bash"),
-        "/bin/cp $ORIG $BACKUP",
-        "ORIG" => "/etc/fstab", "BACKUP" => "/etc/fstab.YaST2.save"
-      )
-
-      fstab = EscapeSpaces(fstab)
-      if !SCR.Write(path(".etc.fstab"), fstab)
+    # Writes the NFS client configuration without
+    # starting/stopping the service.
+    # Autoinstallation uses this and then calls SuSEconfig only once
+    # and starts the services together.
+    # (No parameters because it is too short to abort)
+    # @return true on success
+    def WriteOnly
+      create_mount_points
+      begin
+        write_etc_fstab
+      rescue SystemCallError => err
+        log.error("Error writing /etc/fstab: #{err}")
         # error popup message
         Report.Error(
           _(
             "Unable to write to /etc/fstab.\n" \
-              "No changes will be made to the\n" \
```

Comment:
Hmm, this fix will break all translations... :worried: 

If you really want to fix it then talk to Karl, we could possibly fix it in `en_US` translations.

But as this is an error message displayed in very weird corner cases (where probably the whole system is seriously broken anyway) I would leave it untouched. It's a minor issue anyway, the message is still well understandable.

Reply:
Yes, this will break the translation, so we'll have only the English text for this maintenance update (if there will be any). But I am pretty sure that nobody will notice this when we'll make SLE-12 SP4, so frankly I prefer having the untranslated text here in the rare situation if there is a maintenance update, but have it fixed for the next service pack. I thought about that, actually.

Reply:
OK, the SP4 is a good point!